pkgs/top-level/all-packages.nix

```diff
@@ -39776,7 +39776,7 @@ with pkgs;
 
   opkg-utils = callPackage ../tools/package-management/opkg-utils { };
 
-  OSCAR = qt5.callPackage ../applications/misc/OSCAR { };
+  oscar = qt5.callPackage ../applications/misc/oscar { };
```

Comment:
You can still move package to `by-name` directory.

Reply:
check pkgs/by-name does not allow different callPackages

Reply:
I now replaced the `qt5.callPackage` with `qt5` as an argument in the `package.nix`
this makes overriding specific qt dependencies harder, I think
but otherwise it works

Reply:
To clarify, my suggestion was meant to move location of files, not how you are calling it at `all-packages.nix`.